sections/shared/Layout/SideNav/SideNav.tsx

```diff
@@ -20,17 +21,25 @@ import { CryptoCurrency, Synths } from 'constants/currency';
 import { SIDE_NAV_WIDTH, zIndex } from 'constants/ui';
 
 import { MENU_LINKS, MIGRATE_MENU_LINKS } from '../constants';
+import SubMenu from './SubMenu';
 import PriceItem from './PriceItem';
 import PeriodBarStats from './PeriodBarStats';
 import BalanceItem from './BalanceItem';
 import CRatioBarStats from './CRatioBarStats';
 
+const getKeyValue = <T extends object, U extends keyof T>(obj: T) => (key: U) => obj[key];
```

Comment:
To fix a TS error with objects and indices.

Reply:
I wonder if it's because `menuLinkItemRefs` and `subMenuConfiguration` need to be typed

Reply:
Yeah I think so too.